### PR TITLE
Start fixing unicode handling for Python3

### PR DESCRIPTION
**Pull request by ashahba**
fixes #{issue_num_here_if_applicable}

If you aren't done, please add the `wip` label until you're ready for review.

## Reviewer Checklist

 - [ ] Do you understand it?
 - [ ] Is it correct?
 - [ ] Is it safe?
 - [ ] Is it legally compliant?
 - [ ] Is it robust?
 - [ ] Is it simple?
 - [ ] Is it tested?
 - [ ] Is it documented?
 - [ ] Is the style consistent?

See [the reviewer guide](docs/reviews.md) for more detail on each check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # mlt
 Machine Learning Container Tool
 
+> MLT: it’s like the keras of kubernetes
```
Comment (nqn) [inline README.md]: The '’' is causing the issue?

**Comment by ashahba** [comment]
@nqn this is good for now, but I was thinking more of enabling `unicode` handling in general for `Python3`, but I guess we worry about that when it becomes an issue 🙂

**Comment by nqn** [comment]
@ashahba Can we have this done in Jason's PR?

**Comment by ashahba** [comment]
@nqn absolutely! let me push this to his branch and close this one.

**Comment by ashahba** [comment]
This is being addressed by #178

**Comment by ashahba** [comment]
Closing this now.

**Pull request closed without merging**